superset-frontend/src/components/Checkbox/index.tsx

```diff
@@ -26,22 +30,16 @@ interface CheckboxProps {
 
 export default function Checkbox({ checked, onChange, style }: CheckboxProps) {
   return (
-    <span style={style}>
-      <i
-        role="button"
-        tabIndex={0}
-        className={`fa fa-check ${
-          checked ? 'text-primary' : 'text-transparent'
-        }`}
-        onClick={() => {
-          onChange(!checked);
-        }}
-        style={{
-          border: '1px solid #aaa',
-          borderRadius: '2px',
-          cursor: 'pointer',
-        }}
-      />
+    <span
+      style={{ verticalAlign: 'top', ...style }}
```

Comment:
We could probably use emotion to make this `<span...` a `<Styles...` tag, using Emotion, to address this alignment and the piecemeal ones added to the SVG files, like so:

```
const Styles=styled.span`
  vertical-align: top;
  ...
  svg {
    vertical-align: top;
  }
`
```